src/mirakuru/base.py

```diff
@@ -226,7 +226,10 @@ def _kill_all_kids(self, sig):
         pids = processes_with_env(ENV_UUID, self._uuid)
         for pid in pids:
             log.debug("Killing process %d ...", pid)
-            os.kill(pid, sig)
+            try:
+                os.kill(pid, sig)
+            except ProcessLookupError:
```

Comment:
Could you add a comment here? I think it'll be useful here if and when mirakuru will be undergoing refactoring :)